plugins/lookup/github_app_access_token.py

```diff
@@ -84,8 +89,10 @@
 display = Display()
 
 
-def read_key(path):
+def read_key(path, private_key=None):
     try:
+        if private_key:
+            return jwk_from_pem(private_key.encode())
```

Comment:
If neither are passed by the user, they will get an error trying open the file `None` or something like that. I believe it would be better to have a validation check telling the user they must pass one or the other.

Reply:
Done in [8e928bb](https://github.com/ansible-collections/community.general/pull/8989/commits/8e928bbbcbffb7dff967d6daf147d434175926d1)